### PR TITLE
Initial implementation of metrics -> BigQuery loader

### PR DESCRIPTION
**Pull request by justinsb**

### CODE REVIEW

File: metrics/access-logs/create-schema.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -e
+set -x
+
+PROJECT=`gcloud config get-value project`
+
+# Must generally create in same location as GCS buckets, doesn't apply to US but still a good idea!
+# https://cloud.google.com/bigquery/docs/batch-loading-data#data-locations
+bq mk --location=US --dataset ${PROJECT}:kubernetes_public_logs
```
Comment (ameukam) [inline metrics/access-logs/create-schema.sh:10]: @justinsb can we use this project and this dataset : https://github.com/kubernetes/k8s.io/pull/2031 ?

**Comment by spiffxp** [comment]
Is this used or intended for merge? I'm going to close next week if not.

ref: https://github.com/kubernetes/k8s.io/issues/1834

**Comment by BobyMCbobs** [comment]
This was a good bunch of work for loading the data in in. 
In the end, we took the route of building out a job that uses Postgres that pulls in ASN data from this repo and several other sources. 

https://github.com/ii/org/blob/afc91d7/research/asn-data-pipeline/asn_pipeline_docker_file.org

**Pull request closed without merging**